kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -270,7 +281,7 @@
                           :aria-controls="`question-panel-${question.question_id}`"
                           @click="toggleItemState(question.question_id)"
                         >
-                          <span>{{ title }}</span>
+                          <span>{{ title + " " + question.counter_in_exercise }}</span>
```

Comment:
Should question.counter_in_exercise be wrapped in ()?

Reply:
I think that would be consistent with the previous implementation?